### PR TITLE
Add test to see if /dev/shm is working correctly

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by rhatdan** [comment]
There is a chance that /dev/shm was working correctly in 1.9, adding this test to see if it was.

File: test/image.bats [hunk 1]
```diff
@@ -136,6 +136,7 @@ function teardown() {
 }
 
 @test "image pull with signature" {
+	skip "registry has some issues"
```
Comment (TomSweeneyRedHat) [inline test/image.bats]: It's too bad there's not a "Skip with warning" type of skip.  It would be nice to run this in general.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
Actually if this tests passes travis then we don't have the breakage in V1.9 only on newer versions.  We could choose to add the tests or close the PR.

**Comment by mrunalp** [comment]
I am sure that the breakage is because of the newer runtime-tools. We can add the tests anyways.

**Comment by mrunalp** [comment]
@rhatdan Can we cherry-pick the two commits separately in this PR? That way it becomes easier to untangle and backport stuff to the 1.9 branch.

**Comment by mrunalp** [comment]
We missed the /dev/shm -> /etc/shm typo in this PR to master in the update to runtime-tools - https://github.com/kubernetes-incubator/cri-o/commit/23d20c9db54147e83f43b124237858ddd9cc55f0

**Pull request merged**